docker/docker-windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package docker
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var (
+	errDockerMachineUnknown = fmt.Errorf("unknown DOCKER_MACHINE_NAME")
+)
+
+func (r *Runner) ProjectDir() string{
+	return "/c/Users/dev/GoPath/src/github.com/evo-cloud/hmake"
```

Comment:
Shall we return error immediately if user's project dir is not in `C:\Users`? It fails fast and tells user why it can't run.